micro-infra-hystrix/src/main/groovy/com/ofg/infrastructure/hystrix/IsHystrixServletEnabled.groovy

```diff
@@ -0,0 +1,20 @@
+package com.ofg.infrastructure.hystrix
+
+import com.ofg.infrastructure.config.PropertyAbsentOrEnabledCondition
+import groovy.transform.CompileStatic
+import groovy.transform.PackageScope
+
+/**
+ * Checks for the presence of the property to verify whether Hystrix is enabled or not
+ */
+@CompileStatic
+@PackageScope
+class IsHystrixServletEnabled extends PropertyAbsentOrEnabledCondition {
+
+    public static final String HYSTRIX_SERVLET_ENABLED = 'com.ofg.infra.microservice.hystrix.servlet'
```

Comment:
One configuration key is named `hystrix.stream-servlet.path` and the other `com.ofg.infra.microservice.hystrix.servlet`. I suggest renaming this one to: `hystrix.stream-servlet.enabled`